ECommerceAPI/ECommerceAPI.csproj

```diff
@@ -9,15 +9,15 @@
   <ItemGroup>
     <PackageReference Include="AutoMapper" Version="13.0.1" />
     <PackageReference Include="ErrorOr" Version="2.0.1" />
-    <PackageReference Include="Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore" Version="8.0.4" />
+    <PackageReference Include="MailKit" Version="4.6.0" />
```

Comment:
why you need? you can use third party for email services like SendGrid

Reply:
For email messages, 
Today I tried to learn how to use SendGrid but I found some problems, such as this message
 "We are unable to log you in at this time, please check your email for further information."
 that appears when I want to log into the account
I'll try again later